dsautils/cli.py

```diff
@@ -41,33 +41,32 @@ def radecel(mjd=None, localtime=None, utctime=None):
     """
 
     if mjd is not None and localtime is None and utctime is None:
-        tu = int(1000*Time(mjd, format='mjd').unix)
+        tu = int(MS_PER_SECOND*Time(mjd, format='mjd').unix)
```

Comment:
I should have spent a bit more time on this section. The nearly duplicated 3 lines of code in each if,else block only differ by the time and format parameter and could be converted into a function. Then each if block uses this function. The 1000 -> MS_PER_SECOND would have only needed to be replaced once.